src/pip/_internal/utils/misc.py

```diff
@@ -678,9 +679,14 @@ def call_subprocess(cmd, show_stdout=True, cwd=None,
         env.pop(name, None)
     try:
         proc = subprocess.Popen(
-            cmd, stderr=subprocess.STDOUT, stdin=subprocess.PIPE,
-            stdout=stdout, cwd=cwd, env=env,
-        )
+            cmd,
+            stdin=subprocess.PIPE,
+            stdout=stdout,
+            stderr=subprocess.STDOUT,
+            cwd=cwd,
+            env=env)
```

Comment:
Why is the above being reformatted? Is this pip's recommended style? It doesn't look like there is any change, and the previous formatting seems nicer, at least to me.

Reply:
Apologies - this was a bad rebase on my part. My original patch added `stdin=subprocess.PIPE`, and I reformatted as part of that. But since the patch went into review, 1c36fbc6c13cf1a8998ad0ebfeb9022e3508c18b added `stdin=subprocess.PIPE` too, and so my change now amounts to only formatting tweaks. I'll fix this shortly.